src/cmd/check.go

```diff
@@ -114,12 +167,16 @@ func (self *CheckCreateType) resolveCategoryAliases(client *opslevel.Client, use
 		}
 		self.Spec["categoryId"] = category.Id
 	} else {
+		if self.IsUpdateInput() {
```

Comment:
If the request is to update a check and prompt UI isn't being used, if they didn't specify a value for the aliases (like a category or level or integration) don't add that to the request. (This makes the mutation not change that field)